rvs/src/rvsexec.cpp

```diff
@@ -446,7 +449,9 @@ void rvs::exec::do_help() {
   cout << "-a --appendLog     When generating a debug logfile, do not overwrite the content\n";
   cout << "                   of the current log. Use in conjuction with -d and -l options.\n\n";
 
-  cout << "-c --config        Specify the test configuration file to use.\n\n";
+  cout << "-c --config        Specify the test configuration file to use.\n";
```

Comment:
Wrong place for the edit. Its already in -j. Please revert this.

Reply:
done